### PR TITLE
Image tag with version

### PR DESCRIPTION
**Pull request by joseph-bisonai**
# Description

Image tage include package.json version info

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist before requesting a review

- [x] I have performed a self-review of my code.
- [ ] If it is a core feature, I have added thorough tests.

## Deployment

- [ ] Should publish npm package
- [ ] Should publish Docker image

### CODE REVIEW

File: .github/workflows/api.image+upload.yaml [hunk 1]
```diff
@@ -62,5 +69,5 @@ jobs:
 
       - name: Publish Image to ECR(orakl)
         run: |
-          docker tag orakl-api ${{ env.ecr_url }}:${{ needs.prepare.outputs.tag_date }}.${{ needs.prepare.outputs.tag_git_hash }}
-          docker push ${{ env.ecr_url }}:${{ needs.prepare.outputs.tag_date }}.${{ needs.prepare.outputs.tag_git_hash }}
\ No newline at end of file
+          docker tag orakl-api ${{ env.ecr_url }}:${{ needs.prepare.outputs.version }}.${{ needs.prepare.outputs.tag_date }}.${{ needs.prepare.outputs.tag_git_hash }}
```
Comment (martinkersner) [inline .github/workflows/api.image+upload.yaml:72]: I think you are missing `V`.

```suggestion
          docker tag orakl-api ${{ env.ecr_url }}:V${{ needs.prepare.outputs.version }}.${{ needs.prepare.outputs.tag_date }}.${{ needs.prepare.outputs.tag_git_hash }}
```

File: .github/workflows/api.image+upload.yaml [hunk 2]
```diff
@@ -62,5 +69,5 @@ jobs:
 
       - name: Publish Image to ECR(orakl)
```
Comment (martinkersner) [inline .github/workflows/api.image+upload.yaml:70]: I think there should be `API` instead of `orakl`.

**Review by martinkersner — APPROVED** [review]
LGTM! There are two things that should be fixed before merging.

File: .github/workflows/api.image+upload.yaml (hunk 1 above)
Comment (joseph-bisonai) [reply .github/workflows/api.image+upload.yaml]: I add 'V'

**Pull request merged**